protostar/commands/test/test_result_structured_formatter.py

```diff
@@ -0,0 +1,193 @@
+from typing import Callable
```

Comment:
Why don't you use `StructuredMessage`?

Reply:
[wrap in the StructuredMessage](https://github.com/software-mansion/protostar/pull/1254/commits/e2c3d62e4693b96fe89949030c488c378772caf2)